src/Squirrel/UpdateManager.ApplyReleases.cs

```diff
@@ -159,7 +159,13 @@ public async Task FullUninstall()
                     Utility.PackageDirectoryForAppDir(rootAppDirectory),
                     thisRelease.Filename));
 
-                var exePath = Path.Combine(Utility.AppDirForRelease(rootAppDirectory, thisRelease), exeName);
+                var releaseDir = Utility.AppDirForRelease(rootAppDirectory, thisRelease);
```

Comment:
We seem to duplicate this block of code three times. Can we extract this into a utility method?